src/schema/sources.ts

```diff
@@ -1231,10 +1232,9 @@ export const resolvers: IResolvers<unknown, BaseContext> = traceResolvers<
     ): Promise<GQLSourceCategory> =>
       graphorm.queryOneOrFail(ctx, info, (builder) => ({
         ...builder,
-        queryBuilder: builder.queryBuilder.where(
-          `"${builder.alias}"."id" = :id`,
-          { id },
-        ),
+        queryBuilder: builder.queryBuilder
+          .where(`CAST("${builder.alias}".id AS text) = :id`, { id })
```

Comment:
You don;t need cast here anymore?

Reply:
We still need it. When the value it compared against is not UUID. It would throw an error.

Reply:
Oh right it's to not fail on slugs :) no problem then